parm/soca/obs/obs_list.yaml

```diff
@@ -15,6 +15,7 @@ observers:
 # Ice concentration
 - !INC ${OBS_YAML_DIR}/icec_amsr2_north.yaml
 - !INC ${OBS_YAML_DIR}/icec_amsr2_south.yaml
+#- !INC ${OBS_YAML_DIR}/icec_mirs.yaml
```

Comment:
sorry for catching this just now @apchoiCMD , but the file names above do not fit our convention, see how sst is constructed: {variable}_{instrument}_{satellite}_{level}.yaml
We'll need to make these changes to the AMSR2 yaml as well but in a different PR.

Reply:
Good point, I will modify accordingly-

Reply:
sorry, I'm just realizing that the markdown rendering didn't do what I expected.
We should follow the convention we have for the sst yaml's, for example:
`sst_avhrr_ma_l3u.yaml`
variable_instrument_satellite_level.yaml

Reply:
made changes and could you please take a look?